modules/mpv-player/android/src/main/java/expo/modules/mpvplayer/MPVLayerRenderer.kt

```
@@ -249,21 +258,68 @@ class MPVLayerRenderer(private val context: Context) : MPVLib.EventObserver {
+            // force-window=yes, mpv tears down the decoder at stop time but
+            // tries to keep the VO alive — which fires an internal
+            // video-reconfig. On libmpv 1.0's gpu-next/android backend that
+            // reconfig path crashes with "Missing surface pointer" because we
+            // detach the Surface below before mpv's worker reaches the
+            // reconfig step (command() is async). Setting force-window=no
+            // first makes mpv tear VO down cleanly instead of attempting a
+            // doomed re-init, eliminating the fatal VO error and the
+            // "playback won't restart" aftermath.
+            try {
+                m.setOptionString("force-window", "no")
+            } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)